packages/console/src/pages/ConnectorDetails/index.tsx

```diff
@@ -109,6 +109,7 @@ const ConnectorDetails = () => {
                 setIsReadMeOpen(false);
               }}
             >
+              {/* TODO - Implement the content when the documentation website is ready. */}
```

Comment:
connector's readme is different from others, is already ready for FE, so this line of comment should be removed.